include/aikido/planner/dart/ConfigurationToEndEffectorOffset.hpp

```diff
@@ -16,27 +17,85 @@ namespace dart {
 class ConfigurationToEndEffectorOffset : public Problem
```

Comment:
We should augment this documentation here with the types of constructors that are supported.